doc/release-notes.rst

```diff
@@ -7684,6 +7684,93 @@ Notable Changes
 * rocksdb: update to latest (Xiaoxi Chen)
 * rpm: loosen ceph-test dependencies (Ken Dreyer)
 
+v0.94.10 Hammer
+===============
+
+This Hammer point release fixes several bugs and adds two new features:
+
+Feature No. 1: ceph-objectstore-tool and ceph-monstore-tool now enable user to
+rebuild the monitor database from OSDs. (This feature is especially useful when
+all monitors fail to boot due to leveldb corruption.)
+
+Feature No. 2: RADOS Gateway off-line resharding command.
```

Comment:
An offline tool to reshard a bucket's index. 
Usage:
$ radosgw-admin bucket reshard --bucket= --num-shards=<num_shards>

This will create a new linked bucket instance that points to the newly created index objects. The old bucket instance still exists and currently it's up to the user to manually remove the old bucket index objects.